docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -116,3 +117,36 @@ categories for which the predicted probabilities are reported is 2
 (`num_top_classes`). The result is written to the `prediction` field and the top 
 classes to the `probabilities` field. Both fields are contained in the 
 `target_field` results object.
+
+
+[discrete]
+[[inference-processor-feature-importance]]
+==== {feat-imp-cap} object mapping
+
+Update your index mapping as you can see below to get the full benefit of 
+aggregating and searching for 
+{ml-docs}/dfa-classification.html#dfa-classification-feature-importance[{feat-imp}].
+
+[source,js]
+--------------------------------------------------
+{
```

Comment:
This should indicate that this is the mapping for the feature importance result field.